_layouts/services.html

```diff
@@ -142,6 +142,9 @@
     padding-right: 2em;
     padding-bottom: 1em;
   }
+  .flex-row:not(.top) section p, .flex-row:not(.top) section ul {
+    margin-left: 75px;
+  }
```

Comment:
This feels like a hacky way to do this, instead of doing it structurally.

Really you want a grid with these boxes (or something similar depending on exactly what you want):
![grid-lay](https://github.com/user-attachments/assets/8701116a-3689-4bf8-92c0-d411013f8ad7)

See https://css-tricks.com/snippets/css/complete-guide-grid/ for how to use grid

You can use media queries to change the grid layout for different screen sizes and get the same effect.

The difference is that margins are extremely fragile and may collapse and interact with other things in unintuitive ways, whereas grid does exactly what you want.

Reply:
Fixed in [05a88e8](https://github.com/derangeddk/deranged.dk/pull/114/commits/05a88e8e82793d6b3a38dff548bc1ec74ba183dd)